ome-files/downloads/index.html

```diff
@@ -5,6 +5,7 @@
         <div class="callout large primary" id="bg-image-ome-files">
             <div class="row column text-center">
                 <h1>OME Files Downloads</h1>
+                <a href="https://docs.openmicroscopy.org/latest/ome-files/" target="_blank" class="hero-link">Read the Docs</a>
```

Comment:
This link doesn't actually work cc @sbesson

Reply:
We definitely need to sort out the documentation/API URLs for the upcoming OME Files 0.5.0 release and unify with the rest of the project. I will start a document listing the various elements so that we can discuss on a concrete proposal. For now, a workaround would be to add the documentation bundle to the downloads page which contains a hierarchical version of all the documentation.